notebooks/langchain/langchain-vector-store-using-elser.ipynb

```diff
@@ -86,11 +86,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 29,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "url = \"https://raw.githubusercontent.com/elastic/elasticsearch-labs/main/example-apps/workplace-search/example-data/data.json\"\n",
+    "url = \"https://raw.githubusercontent.com/elastic/elasticsearch-labs/main/example-apps/workplace-search/data/data.json\"\n",
```

Comment:
This change is not related to this PR. But thought of adding it as the notebook was using wrong path.